shell/platform/embedder/embedder.h

```diff
@@ -31,6 +31,119 @@ typedef enum {
   kSoftware,
 } FlutterRendererType;
 
+typedef enum {
+  // Indicate there is a running accessibility service which is changing the
+  // interaction model of the device.
+  kFlutterAccessibilityFeatureAccessibleNavigation = 1 << 0,
+  // Indicate the platform is inverting the colors of the application.
+  kFlutterAccessibilityFeatureInvertColors = 1 << 1,
+  // Request that animations be disabled or simplified.
+  kFlutterAccessibilityFeatureDisableAnimations = 1 << 2,
+  // Request that text be rendered at a bold font weight.
+  kFlutterAccessibilityFeatureBoldText = 1 << 3,
+  // Request that certain animations be simplified and parallax effects
+  // removed.
+  kFlutterAccessibilityFeatureReduceMotion = 1 << 4,
+} FlutterAccessibilityFeature;
+
+// The set of possible actions that can be conveyed to a semantics node.
+typedef enum {
```

Comment:
Cab there be some reuse with https://github.com/flutter/engine/blob/6c68bf3caf89ea3cf9a17e52437a9937054e25a2/lib/ui/semantics/semantics_node.h#L20?

Reply:
@chinmaygarde thoughts on this and the others?

I'd love to kill off the duplication of these enums but we can't use the internal one in the embedder API, and architecturally it feels a bit dirty to have the internal stuff depend on an embedder-specific API, though admittedly the embedder API should be the canonical embedder eventually.

Reply:
(If not, there should probably at least be a comment on them to keep them in sync with the canonical list in the dart file, see comments in the files I linked to)

Reply:
Added the comment.

Reply:
I think it is fine for the internal stuff to use the decls in the embedder.h header and prevent reuse. But your call.

Reply:
sgtm. I'll make those changes in a follow-up. That saves one case of duplication anyway.